src/main/java/com/blackducksoftware/integration/jira/blackduck/BlackDuckAssignUtil.java

```diff
@@ -32,9 +32,9 @@
 import java.util.stream.Collectors;
 
 import org.apache.commons.lang3.StringUtils;
-import org.apache.log4j.Logger;
+import org.slf4j.Logger;
```

Comment:
Have we confirmed that we can get these logs? Do we know where they are? Do we need to update any documentation?

Reply:
The logging appears in the console and there are no class path issues when logging. So I imagine it works. Based off of the way slf4j works, I believe that we still use log4j as it's the one on the classpath. It's just now wrapped by slf4j.